src/components/RouteUpdate.tsx

```diff
@@ -372,6 +398,28 @@ const RouteUpdate = () => {
                                     <InputNumber min={1} max={9999} autoComplete="off"/>
                                 </Form.Item>
                             </Col>
+                            <Col span={24}>
+                                <Form.Item
+                                    name="groups"
+                                    label="Distribution groups"
+                                    tooltip="Distribution groups define to which group of peers this route will be distributed to"
```

Comment:
Maybe like that?

`NetBird will advertise this route to peers that belong to the following groups`